FEM/rf_pcs.cpp

```diff
@@ -3371,6 +3375,10 @@ void CRFProcess::VariableDynamics()
 	pcs_secondary_function_name[pcs_number_of_secondary_nvals] = "STRAIN_PLS";
 	pcs_secondary_function_unit[pcs_number_of_secondary_nvals] = "-";
 	pcs_secondary_function_timelevel[pcs_number_of_secondary_nvals] = 1;
+	pcs_number_of_secondary_nvals++;
```

Comment:
Why all these if conditions? There's no if condition for STRAIN_PLS either, although that's specific to some models only. Also for principal stress and direction output there's no such condition.